### PR TITLE
Fix script compat with older node for testing on `donna`

### PR DESCRIPTION
**Pull request by fw-immunant**

### CODE REVIEW

**Review by kkysen — COMMENTED** [review]
Can we just install a newer `node`?  I just used [`volta`](https://docs.volta.sh/guide/getting-started):

```sh
curl https://get.volta.sh | bash
volta install node@latest
```

**Comment by kkysen** [comment]
Node 12, which is the distro version on donna, is end of life as of April 2022.

**Comment by fw-immunant** [comment]
> Node 12, which is the distro version on donna, is end of life as of April 2022.

You're right--apparently other folks are also upset about the Ubuntu 2022 LTS having an out-of-date Node version: https://bugs.launchpad.net/ubuntu/+source/nodejs/+bug/1861327

But Ubuntu LTS isn't a rare configuration either, and it doesn't hurt us much to deal with this distro foible.

File: scripts/pretty-instrument-err.mjs [hunk 1]
```diff
@@ -30,8 +30,8 @@ stderr = stderr.split("\n")
             };
         }
     })
-    .filter(e => e.level?.includes("error"))
-    .map(e => e.rendered ?? e.message)
+    .filter(e => e.level ? e.level.includes("error") : false)
+    .map(e => e.rendered ? e.rendered : e.message)
```
[the remark this answers is not in the document]
Comment (kkysen) [reply scripts/pretty-instrument-err.mjs]: Adjusted and commented the ?. and ?? backports to ensure they're equivalent on falsy values in https://github.com/immunant/c2rust/pull/472/commits/d67499410f1f6cb287e9a5b54c5ca3ea815ddf68.

**Pull request merged**